static/app/views/performance/platformCompatibilityChecker.tsx

```diff
@@ -0,0 +1,136 @@
+import {useEffect, useMemo} from 'react';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t, tct} from 'sentry/locale';
+import type {PageFilters} from 'sentry/types';
+import {useDiscoverQuery} from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {usePageAlert} from 'sentry/utils/performance/contexts/pageAlert';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import usePrevious from 'sentry/utils/usePrevious';
+import useProjects from 'sentry/utils/useProjects';
+
+const MAX_PROJECTS_TO_LIST = 3;
+
+function getEventView(selection: PageFilters, compatibleSDKNames: string[]) {
+  const {projects: selectedProjectIds} = selection;
+
+  return EventView.fromNewQueryWithPageFilters(
+    {
+      dataset: DiscoverDatasets.SPANS_INDEXED,
+      fields: ['project_id', 'sdk.name', 'count()'],
+      projects: selectedProjectIds,
+      query: `has:sdk.name !sdk.name:[${compatibleSDKNames.join(',')}]`,
+
+      name: '',
+      version: 2,
+    },
+    selection
```

Comment:
🤔 We can't determine the sdk name from the project endpoint?

Reply:
I've been advised not to trust the project platform data because people supposedly misconfigure their project platform, or use multiple SDKs to send into a single project, so it's a better source of truth to query for what the data says. Although it is slower :(